recipe/meta.yaml

```diff
@@ -32,7 +32,7 @@ requirements:
     # when this is changed also update bld.bat
     - hdf5 1.10.1
     - six
-    - unittest2    # [py26 and py27]
+    - unittest2    # [py26 or py27]
```

Comment:
:sheep:  "I want it for 2.6 and 2.7"

Reply:
Indeed. Probably the sort of thing that got Knuth thinking about Literate programming. :)